app/services/preserved_object_handler.rb

```diff
@@ -112,13 +112,42 @@ def confirm_version
     results
   end
 
+  # If validation doesn't pass then create with Status.invalid - so need another flag to tell create_db_object -
+  # can jiust call that right?
+
+  def update_version_with_validation
```

Comment:
Sarav - we need to confer about this because this is EXACTLY what I was working on.  I thought ticket #285 was a very simple "we discover the object doesn't yet exist, so we called `create`" and substitute `create_after_validation` ... but this is essentially doing ticket #283, which I had started.

Maybe my assumption about "update_version" actually calling create when the druid wasn't in the catalog was mistaken, which would have made #285 very confusing ...